doc/creating_commands.md

```diff
@@ -100,11 +100,14 @@ Option declaration is the same as in clamp so please read it's
 on that topic. However, unlike in Clamp, the option accessors in Hammer are created with prefix 'option_', to avoid
 conflict with methods of the commands. So to access value of an `--name` option you have to call `option_name()`
 
+Also Hammer offers predefined options now. To define them in your command use `use_option :option_name` method.
+[Here](https://github.com/theforeman/hammer-cli/blob/master/lib/hammer_cli/options/predefined.rb#L7) is the list of predefined options.
 
 Example option usage could go like this:
 ```ruby
 class HelloCommand < HammerCLI::AbstractCommand
 
+  use_option :option_name, :or_names
```

Comment:
@ofedoren Here I'd expect naming conflict with the option as both could be called `--name`, also from the example output it is not apparent what it does. There is just one --name option in the output. Maybe it would be better not to mention `use_option` here and create separate subsection  `####Predefined options` next to `#### Deprecated options`. May make sense to explain how it works, how to use it and list available predefined options. With `:fields` we can add brief description with reference to `Printing hash records` section for more details.